src/hipoc/hipoc_program.cpp

```diff
@@ -232,6 +233,8 @@ struct HIPOCProgramImpl
     void
     BuildCodeObjectInFile(std::string& params, const std::string& src, const std::string& filename)
     {
+        MIOPEN_LOG_I("BuildCodeObjectInFile for file: " << filename);
```

Comment:
Can we make these I2 to reduce unnecessary verbosity

Reply:
I will get rid of those. This is some useful development logs, but is redundant in production code.